letsdrill.rb

```diff
@@ -1,12 +1,23 @@
 def get_letter_grade(integer)
+    case integer
+        when (0..59)
+             'F'
+        when (60..69)
+             'D'
+        when (70..79)
+             'C'
+        when (80..89)
+             'B'
+        else
+            'A'
+        end
+ 
  
-  #Put your code here!
-
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+array.min_by {|x| x.length}
 
```

Comment:
Cool, very simple, I like it. One tiny thought: would `array.min_by &:length` work?